src/Development/Shake/Internal/Core/Types.hs

```diff
@@ -60,8 +60,14 @@ import Prelude
 --   To raise an exception call 'error', 'fail' or @'liftIO' . 'throwIO'@.
 --
 --   The 'Action' type is both a 'Monad' and 'Applicative'. Anything that is depended upon applicatively
---   will have its dependencies run in parallel. For example @'need' [\"a\"] *> 'need [\"b\"]@ is equivalent
---   to @'need' [\"a\", \"b\"]@.
+--   will have its dependencies run in parallel. For example,
+--   @'Development.Shake.Internal.Rules.File.need' [\"a\"] '*>' 'Development.Shake.Internal.Rules.File.need' [\"b\"]@
```

Comment:
I needed to fully qualify the path to `need` here, since it appears that Haddock can't figure it out:

https://hackage.haskell.org/package/shake-0.19.8/docs/Development-Shake.html#t:Action

![image](https://github.com/ndmitchell/shake/assets/64804/d9521437-a839-4e2c-87fc-31a6f41ccc6b)

See how the `need`s on the last line here aren't actually links, since Haddock doesn't seem to know where it should link to.   Writing these as fully-qualified references should fix this.